src/main/java/de/ait/secondlife/services/mapping/OfferMappingService.java

```diff
@@ -0,0 +1,30 @@
+package de.ait.secondlife.services.mapping;
+
+import de.ait.secondlife.domain.dto.OfferCreationDto;
+import de.ait.secondlife.domain.dto.OfferResponseDto;
+import de.ait.secondlife.domain.entity.Offer;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+
+@Mapper(componentModel = "spring")
+public interface OfferMappingService {
+
+
+    @Mapping(source = "userId", target = "ownerId")
+    @Mapping(source = "status.id", target = "statusId")
+    @Mapping(source = "winnerBid.id", target = "winnerBidId")
+    @Mapping(target = "endAt", expression = "java(offer.getCreatedAt().plusDays(offer.getAuctionDurationDays()))")
```

Comment:
endAt will be set up after offer verification

Reply:
This is a temporary solution. After the codes are merged, it is necessary to make a task for it.